lib.py

```diff
@@ -1,12 +1,35 @@
 from __future__ import annotations
 
+import os
+from abc import ABC
+from abc import abstractmethod
 from functools import cache
+from subprocess import CalledProcessError
+from subprocess import run
+
+
+class MetaCheck(ABC):
+    tags: set
```

Comment:
What is? There's no doc, here.

Reply:
It is meant to allow us to mark checks with specific tags so we can, for instance, not run multiple checks that try to edit the same file at once, or being able to only run checks for a specific tag. With that said, we should add some documentation around that.